client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -52,7 +54,13 @@ export const ReportArgumentsModal: FC<ReportArgumentsModalProps> = ({
       }
     >
       <JsonForm
-        data={data}
+        data={{
+          monthlyConsumptionLookBackPeriod:
+            store?.preferences.monthlyConsumptionLookBackPeriod,
+          monthsOverstock: store?.preferences.monthsOverstock,
+          monthsUnderstock: store?.preferences.monthsUnderstock,
+          ...(data as object),
+        }}
```

Comment:
Super hacky... and doesn't affect other reports... but still very hacky >.>

Reply:
this is my suggestion:

```
  const [data, setData] = useState<{} | undefined | null>({});

...

  const {
    monthlyConsumptionLookBackPeriod,
    monthsOverstock,
    monthsUnderstock,
  } = store?.preferences ?? {};

  const jsonData = {
    monthlyConsumptionLookBackPeriod,
    monthsOverstock,
    monthsUnderstock,
    ...data,
  };

...

        data={jsonData}
```